dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/IconService.java

```diff
@@ -110,12 +110,15 @@ public interface IconService {
    * @param key the key of the icon to update
    * @param description the new icons description
    * @param keywords the new icons keywords
+   * @param isCustom Icon is CustomIcon or default.
```

Comment:
can we follow the same structure when writing comments? 
for instance, in the lines above, Icon is written in lower case and CustomIcon is written in camel case but default is not.
If we follow the previous pattern, it should be something like this:
`* @param isCustom icon is custom or default`

Reply:
resolved